# Properties of a control group

string group_name
string type
uint32[] active_units